templates/vue-starter-template/app/utils/validation/rules/personalDataFormRules.ts

```diff
@@ -0,0 +1,27 @@
+import { customValidators } from "@@/i18n/utils/i18n-validators";
+import { type Ref, computed } from "vue";
+
+export default function (accountType: Ref<string>) {
```

Comment:
what about making it named export? better for tree shaking and gives us opportunities to group rules

Reply:
sure, why not